.github/workflows/post-merge.yaml

```diff
@@ -0,0 +1,42 @@
+name: post-merge
+
+on:
+  # Run on merges to master or tag pushes
+  push:
+    branches: [ 'master' ]
+    tags: [ '*' ]
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  lambda-test:
+    uses: "./.github/workflows/test.yaml"
```

Comment:
do we even need this `lambda-test` job here.  would it work if we just moved this step to the `package-and-publish` job?

Reply:
yeah, we do need this because this is calling our reusable workflow as a job; in order to call something as a step in another job it needs to be defined as an action, not a workflow (like with `sam-build`).

Reply:
ahh, good to know.